### ISSUE TITLE
Readme incorrect. `./main` -> `./main run`

### ISSUE THREAD

**Issue by sonusainier** [nanoscopic/controlfloor#27]
<img width="1314" alt="Screenshot 2021-05-24 at 11 01 58 AM" src="https://user-images.githubusercontent.com/83640445/119301093-8dc07180-bc7f-11eb-8e2b-3fae863f78bd.png">

**Comment by ababushkin-bscmsc** [comment]
use command "./main run"

**Comment by nanoscopic** [comment]
You are correct the readme still says to just run `./main` and not correctly to run `./main run`. I will update it. Thanks for pointing this out.

**Comment by nanoscopic** [comment]
Meh; am lazy and not maintaining this. Submit a PR for this and I'll happily hit approve though.
